core/reactor.h

```diff
@@ -512,7 +512,11 @@ struct reaction_t {
     void* self;    // Pointer to a struct with the reactor's state. INSTANCE.
     int number;    // The number of the reaction in the reactor (0 is the first reaction).
     index_t index; // Inverse priority determined by dependency analysis. INSTANCE.
+    #ifdef BIT_32
+    unsigned long chain_id; // Binary encoding of the branches that this reaction has upstream in the dependency graph. INSTANCE.
```

Comment:
Is there a reason for changing chain_id to 32 bits? It's not currently used anyway so I think it would be harmless to leave it at 64 bits.

Reply:
Arduino AVR Compiler complains that it doesn't understand long long types, so we need to explicitly change all long long values to long regardless of if we use them or not.

Reply:
Please add a comment to this effect.

Reply:
Done!